lib/handbook/addon/docs/components/loading-indicator/demo-inline-dark/template.hbs

```diff
@@ -0,0 +1,7 @@
+{{#docs-demo as |demo|}}
+    {{#demo.example name='loading-indicator.inline-dark.hbs'}}
+        {{loading-indicator dark=true inline=true}}
```

Comment:
Just a thought: Could be nice to start using angle-bracket notation in our examples:
```hbs
<LoadingIndicator @dark=true @inline=true />
```

Reply:
Oh, you're right.  I'll go ahead and update that :+1: